jaimemendozadev-los-angeles.md

```diff
@@ -0,0 +1,17 @@
+##Los Angeles, CA
+
+### Restaurants
+
++ [Tsujita](https://www.yelp.com/biz/tsujita-la-artisan-noodle-los-angeles-2)
++ [In-N-Out Burger](http://www.in-n-out.com/)
++ [Langer's Delicatessen](http://www.langersdeli.com/)
++ [Cassell's Hamburgers](http://www.cassellshamburgers.com/)
+
+### Places to Go
++ Dodger Stadium
++ [The Getty](https://www.getty.edu/)
++ [Disneyland](https://disneyland.disney.go.com/destinations/disneyland/)
+
```

Comment:
LOL, for a minute I thought you meant where you can go on a hot date. 

Honestly, on a hot day, I'd stay on indoors. If you're on the West Side (by which I mean anything past West of La Brea Ave.), you should be okay being able to walk on the street without sweating too much.

The closer you get to the ocean, the less hot I think you'll be.